lib/share_progress/email_variant.rb

```diff
@@ -0,0 +1,22 @@
+require 'share_progress'
+require 'share_progress/variant'
+
+module ShareProgress
+  class EmailVariant < Variant
+    attr_accessor :email_subject, :email_body
```

Comment:
looks like we can! http://stackoverflow.com/questions/16563093/specify-attribute-list-in-attr-accessor-with-method-call